src/QuickMarcEditor/QuickMarcEditorRows/QuickMarcEditorRows.js

```diff
@@ -74,6 +77,18 @@ const QuickMarcEditorRows = ({ fields }) => {
                   disabled={isDisabled}
                 />
               </div>
+              <div className={styles.quickMarcEditorActions}>
+                {
+                  !IsAvalibleToAddAfter &&
+                    <IconButton
+                      data-test-add-row
+                      icon="plus-sign"
+                      onClick={() => {
```

Comment:
hm, I don't think it's correct, too many functions

please read about mutators etc from final-form